clusterous/defaults.py

```diff
@@ -39,6 +39,9 @@
 taggable_name_max_length = 64       # Arbitrary but ample, keeping in mind AWS keys can be max 127 chars
 
 security_group_name_format = '{0}-sg'
```

Comment:
Perhaps define the label name format strings over here (e.g for public/private security groups). Either way, security_group_name_format isn't being used, and can be deleted.